src/sysusers/sysusers.c

```diff
@@ -61,10 +62,11 @@ typedef struct Item {
 } Item;
 
 static char *arg_root = NULL;
+static bool arg_cat_config = false;
 static const char *arg_replace = NULL;
 static bool arg_inline = false;
 
-static const char conf_file_dirs[] = CONF_PATHS_NULSTR("sysusers.d");
+static char **config_dirs = CONF_PATHS_STRV("sysusers.d");
```

Comment:
btw, quite frankly i'd drop this static var here (and all other programs we have) entirely... There's no point in having it a static variable, we might as well just invoke the macro directly when we invoke config_files(), I think that would be more readable in the end, as `config_dirs` is so much more opaque than `CONF_PATHS_STRV("sysusers.d")`...

Reply:
It's used in more than one place, that's why I kept it, but yeah, dropping it might be better.

Reply:
there's still a variable allocated for this now. I'd probably do this instead:

```c
#define SYSUSERS_CONF_PATHS CONF_PATHS_STRV("sysusers.d")
```

And then just use the macro at the two places we need it. That way we don't allocate any static variable at all